client/app/scripts/superdesk-authoring/views/send-item.html

```diff
@@ -45,7 +44,7 @@ <h3 ng-show="mode !== 'authoring'" translate>Fetch To</h3>
             </ul>
         </div>
         
-        <div sd-toggle-box ng-if="showPublishSchedule()" data-title="Publishing" data-open="true" data-style="dark">
+        <div sd-toggle-box id="publishScheduleTimestamp" ng-if="mode === 'authoring' && showPublishSchedule()" data-title="Publishing" data-open="true" data-style="dark">
```

Comment:
This line and line 35 are very long and require significant horizontal scrolling, I would suggest breaking them into multiple lines.